addons/account/models/account_move.py

```diff
@@ -145,7 +145,7 @@ def _get_default_invoice_incoterm(self):
         default=fields.Date.context_today
     )
     ref = fields.Char(string='Reference', copy=False, tracking=True)
-    narration = fields.Html(string='Terms and Conditions', compute='_compute_narration', store=True, readonly=False)
+    narration = fields.Html(string='Notes', compute='_compute_narration', store=True, readonly=False)
```

Comment:
No, this was only meant for `account.bank.statement.line`, not for `account.move`